tokio/tests/rt_threaded_alt.rs

```diff
@@ -487,6 +487,31 @@ fn max_blocking_threads_set_to_zero() {
         .unwrap();
 }
 
+/// Regression test for #6445.
+///
+/// After #6445, setting `global_queue_interval` to 1 is now technically valid.
+/// This test confirms that there is no regression in `multi_thread_runtime`
+/// when global_queue_interval is set to 1.
+#[test]
+fn global_queue_interval_set_to_one() {
+    let rt = tokio::runtime::Builder::new_multi_thread()
+        .global_queue_interval(1)
+        .build()
+        .unwrap();
+
+    // Perform a simple work.
+    let cnt = Arc::new(AtomicUsize::new(0));
+    rt.block_on(async {
+        for _ in 0..10 {
+            let cnt = cnt.clone();
+            tokio::spawn(async move { cnt.fetch_add(1, Relaxed) })
+                .await
+                .unwrap();
+        }
+    });
+    assert_eq!(cnt.load(Relaxed), 10);
```

Comment:
You are using `.await` on the tasks immediately when spawning, so you run them one-at-the-time. Is that what you intended?

Reply:
Indeed, running tasks in parallel would be better here.